README.md

```diff
@@ -2,18 +2,99 @@
 
 [![Build Status](https://travis-ci.org/nokia-wroclaw/innovative-project-jackdaw.svg?branch=master)](https://travis-ci.org/nokia-wroclaw/innovative-project-jackdaw)
 
-## Run
 
-`docker-compose up`
+## Table of contents
+1. Project goals
+2. Technologies
+3. Functionality of modules
+4. Installation
+5. Contributing
+
+
+## 1. Project goals
+The goal is to create PoC of platform for streaming calculation using Kafka and Docker in microservices architecture. This platform will allow to perform Big Data Calculation in Streaming mode. As a final product there will be created a web application responsible for data visualization.
```

Comment:
"The goal is to create PoC of platform for streaming calculation"
imho replace with
"The goal is to create PoC for platform performing streaming calculation"

Reply:
It was taken from description of the project made by our supervisors so let's it keep that way

Reply:
maybe, but still, this looks more "english" :)